sched/task/task_posixspawn.c

```diff
@@ -327,6 +330,10 @@ int posix_spawn(FAR pid_t *pid, FAR const char *path,
   sinfo("pid=%p path=%s file_actions=%p attr=%p argv=%p\n",
         pid, path, file_actions, attr, argv);
 
+  /* At the moment, the parent task's environment is used */
+
+  envp = env_get();
```

Comment:
why not use the caller's?

Reply:
Because currently that is always NULL. I'll fix it by getting environ in app.

Reply:
Done